### DISCUSSION TITLE
Install reanimated on React-native-windows project?

### DISCUSSION THREAD

**Discussion by AlexSubZero** [software-mansion/react-native-reanimated#6386] [Q&A] [answered]
Hello, is it possible to make reanimated to work on a react native windows app?
I installed the reanimated with `npm install react-native-reanimated.` i also edited my babel.config.js to look like this
```
module.exports = {
  presets: ['module:@react-native/babel-preset'],
  plugins: [
    ...
    'react-native-reanimated/plugin',
  ],
};
```
But when i try to use a library that require Reanimated i get the following error 

>  Error: [Reanimated] `useAnimatedStyle` was used without a dependency array or Babel plugin. Please explicitly pass a dependency array, or enable the Babel plugin.
> For more, see the docs: `https://docs.swmansion.com/react-native-reanimated/docs/guides/web-support#web-without-the-babel-plugin`.

Is there a solution to this or i should look for another way to move forward?

**Comment by MatiPl01** [comment, marked as answer]
Hey!
We currently do not support windows apps. See supported platforms list in here (more details in [docs](https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting/#native-part-of-reanimated-doesnt-seem-to-be-initialized)):

<img width="442" alt="Screenshot 2024-08-08 at 12 02 06" src="https://github.com/user-attachments/assets/4055765a-6e6e-420d-9917-3dc5b0c43db7">

**Reply by AlexSubZero** [reply]
Hello Mateusz,
Thanks for your reply. One last question 😊 , are you planning on supporting windows and if yes, is there any expected date to release? 

Thanks in advance
~Alex

**Reply by MatiPl01** [reply]
Currently, Windows support is not on our roadmap. I cannot tell you the expected date of the release.
I think that windows support is worth considering and we can discuss in our team about adding support in the future.

**Reply by ArksHub** [reply]
Any update on this?

**Reply by MatiPl01** [reply]
Currently not. I don't know if we will add support for Windows as we have much work to do currently and windows is not that commonly used.

**Reply by Choco-milk-for-u** [reply]
Don’t you already use C++ TurboModules? Is it really that difficult to support React Native Windows? (question, not aggression)

**Reply by MatiPl01** [reply]
@Choco-milk-for-u Thank you for your question!

We already use TurboModules, but, unfortunately, there are always some platform differences when it comes to the native code and we currently have other higher priority tasks than testing what works and fixing what doesn't work on windows. I can't say when we will add support for windows, but it won't rather happen in the near future as we haven't started working on it yet.
